From 7e1814df1fab11f7b901713ef06702f4a85abbb5 Mon Sep 17 00:00:00 2001
From: Evgeny Grin <k2k@narod.ru>
Date: Fri, 19 May 2023 17:49:55 +0300
Subject: [PATCH 1/3] configure: add explicit sed detection

Actually SED variable is set automatically by LT_INIT currently.
Having explicit check should help avoiding problems if/when LT_INIT
changed.

Signed-off-by: Karlson2k (Evgeny Grin) <k2k@narod.ru>
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 9199d7193..07c987e00 100644
--- a/configure.ac
+++ b/configure.ac
@@ -71,6 +71,7 @@ dnl ############################################################################
 dnl Check for programs
 dnl ############################################################################
 
+AC_PROG_SED
 PKG_PROG_PKG_CONFIG
 AC_PROG_INSTALL
 AC_PROG_LN_S
-- 
2.40.1.windows.1

